### PR TITLE
[Jetnews] UI polish bugs #458

### PR DESCRIPTION
**Pull request by angeles-bilbao6**
Updated UI components to match current design

https://github.com/android/compose-samples/issues/458

- **Line height in article doesn't match spec**

Before
image

After
image

- **Spacing on article title on hero card on Home screen is off from spec**

Before
image

After
image

- **Heart icon article bottom app bar should be a Like icon**

Before
image

After
image

- **Divider lines missing between Interest list items**

- **Spacing between Interest list items doesn't match spec**

- **Add interest button doesn't match spec**

Before
image

After
image

- **EXTRA: Tab style doesn't match design**

Before
image

After
image

### CODE REVIEW

File: JetNews/app/src/main/java/com/example/jetnews/ui/interests/InterestsScreen.kt [hunk 1]
```diff
@@ -216,16 +216,30 @@ private fun TabContent(
...
     Column {
         TabRow(
-            selectedTabIndex = selectedTabIndex
+            selectedTabIndex = selectedTabIndex,
+            backgroundColor = MaterialTheme.colors.onPrimary,
+            contentColor = MaterialTheme.colors.primary,
+
         ) {
             tabContent.forEachIndexed { index, tabContent ->
+                val colorText = if (selectedTabIndex == index) MaterialTheme.colors.primary else MaterialTheme.colors.onSurface.copy(alpha = 0.8f)
```
Comment (JoseAlcerreca) [inline JetNews/app/src/main/java/com/example/jetnews/ui/interests/InterestsScreen.kt:225]: Long line, not sure why the linter allowed it...

Comment (angeles-bilbao6) [reply JetNews/app/src/main/java/com/example/jetnews/ui/interests/InterestsScreen.kt]: I just updated line 225

**Comment by JoseAlcerreca** [comment]
LGTM, let's wait for another reviewer before mergin

**Review by yrezgui — APPROVED** [review]

**Comment by c5inco** [comment]
Looks like we are good to go on this PR?

**Comment by c5inco** [comment]
Thank you!!

**Comment by JoseAlcerreca** [comment]
Thanks!

**Pull request merged**